src/main/java/io/jenkins/plugins/oidc_provider/config/ClaimTemplate.java

```diff
@@ -77,6 +95,14 @@ public FormValidation doCheckName(@QueryParameter String value) {
             }
         }
 
+        public FormValidation doCheckRequiredEnvVars(@QueryParameter String value) {
+            if (!value.equals(value.toUpperCase())) {
+                return FormValidation.error("Defined environment variables must be in upper case.");
```

Comment:
:thinking: I would probably downgrade this to a warning. Unix variables are case-sensitive. Windows variables are case-insensitive but if you are running the controller on Windows…it would be best for the `env` used at runtime to be a case-insensitive map (as `EnvVars` uses).